docs/graphql/manual/guides/deployment/index.rst

```diff
@@ -12,6 +12,7 @@ Guides: Deployment
   :depth: 1
   :local:
 
+- :ref:`Render One-Click Deploy with Managed PostgreSQL <deploy_render>`
```

Comment:
Can we move this to the bottom of the list:

![image](https://user-images.githubusercontent.com/5722022/77878521-c66de200-7258-11ea-908a-181a7266ead1.png)

Therefore, we can keep the order in which these guides have been created.

Reply:
Done!